### DISCUSSION TITLE
Anaconda jupyter Erno 13 permission denied

### DISCUSSION THREAD

**Discussion by Keerthi240194** [jupyter/notebook#7471] [General]
While launching jupyter i have encountered below error, Please anyone assist with that 

[I 2024-10-01 11:44:19.864 ServerApp] Extension package jupyter_lsp took 0.1156s to import
[W 2024-10-01 11:44:19.865 ServerApp] A `_jupyter_server_extension_points` function was not found in jupyter_lsp. Instead, a `_jupyter_server_extension_paths` function was found and will be used for now. This function name will be deprecated in future releases of Jupyter Server.
[W 2024-10-01 11:44:20.190 ServerApp] A `_jupyter_server_extension_points` function was not found in notebook_shim. Instead, a `_jupyter_server_extension_paths` function was found and will be used for now. This function name will be deprecated in future releases of Jupyter Server.
[I 2024-10-01 11:44:22.747 ServerApp] Extension package panel.io.jupyter_server_extension took 2.5644s to import
[I 2024-10-01 11:44:22.747 ServerApp] jupyter_lsp | extension was successfully linked.
[I 2024-10-01 11:44:22.763 ServerApp] jupyter_server_terminals | extension was successfully linked.
[I 2024-10-01 11:44:22.779 ServerApp] jupyterlab | extension was successfully linked.
[I 2024-10-01 11:44:22.779 ServerApp] notebook | extension was successfully linked.
[I 2024-10-01 11:44:24.039 ServerApp] notebook_shim | extension was successfully linked.
[I 2024-10-01 11:44:24.039 ServerApp] panel.io.jupyter_server_extension | extension was successfully linked.
[I 2024-10-01 11:44:24.123 ServerApp] notebook_shim | extension was successfully loaded.
[I 2024-10-01 11:44:24.133 ServerApp] jupyter_lsp | extension was successfully loaded.
[I 2024-10-01 11:44:24.133 ServerApp] jupyter_server_terminals | extension was successfully loaded.
[I 2024-10-01 11:44:24.133 LabApp] JupyterLab extension loaded from C:\Users\Keerthivasan\anaconda3\Lib\site-packages\jupyterlab
[I 2024-10-01 11:44:24.145 LabApp] JupyterLab application directory is C:\Users\Keerthivasan\anaconda3\share\jupyter\lab
[I 2024-10-01 11:44:24.147 LabApp] Extension Manager is 'pypi'.
[I 2024-10-01 11:44:24.149 ServerApp] jupyterlab | extension was successfully loaded.
[I 2024-10-01 11:44:24.149 ServerApp] notebook | extension was successfully loaded.
[I 2024-10-01 11:44:24.149 ServerApp] panel.io.jupyter_server_extension | extension was successfully loaded.
[I 2024-10-01 11:44:24.149 ServerApp] Serving notebooks from local directory: C:\Users\Keerthivasan
[I 2024-10-01 11:44:24.149 ServerApp] Jupyter Server 2.14.1 is running at:
[I 2024-10-01 11:44:24.149 ServerApp] http://localhost:8888/tree?token=990ea9734476ede147333d5b4abbb58bad1438e50d548441
[I 2024-10-01 11:44:24.149 ServerApp] http://127.0.0.1:8888/tree?token=990ea9734476ede147333d5b4abbb58bad1438e50d548441
[I 2024-10-01 11:44:24.149 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[E 2024-10-01 11:44:24.164 ServerApp] Failed to write server-info to C:\Users\Keerthivasan\AppData\Roaming\jupyter\runtime\jpserver-3460.json: PermissionError(13, 'Permission denied')
Traceback (most recent call last):
File "C:\Users\Keerthivasan\anaconda3\Scripts\jupyter-notebook-script.py", line 10, in 
sys.exit(main())
^^^^^^
File "C:\Users\Keerthivasan\anaconda3\Lib\site-packages\jupyter_server\extension\application.py", line 623, in launch_instance
serverapp.start()
File "C:\Users\Keerthivasan\anaconda3\Lib\site-packages\jupyter_server\serverapp.py", line 3119, in start
self.start_app()
File "C:\Users\Keerthivasan\anaconda3\Lib\site-packages\jupyter_server\serverapp.py", line 3023, in start_app
self.write_browser_open_files()
File "C:\Users\Keerthivasan\anaconda3\Lib\site-packages\jupyter_server\serverapp.py", line 2890, in write_browser_open_files
self.write_browser_open_file()
File "C:\Users\Keerthivasan\anaconda3\Lib\site-packages\jupyter_server\serverapp.py", line 2913, in write_browser_open_file
with open(self.browser_open_file, "w", encoding="utf-8") as f:
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\Keerthivasan\\AppData\\Roaming\\jupyter\\runtime\\jpserver-3460-open.html'

**Comment by RRosio** [comment]
Hi @Keerthi240194 thank you for opening this discussion! This issue has also been previously reported, could try the suggestions in [this comment](https://github.com/jupyter/notebook/issues/7421#issuecomment-2218215149)?
